lib/server.js

```diff
@@ -4,7 +4,7 @@ var Hapi           = require("hapi");
 var Bluebird       = require("bluebird");
 var MongoMapper    = require("genesis").MongoMapper;
 
-var mapper = new MongoMapper(process.env.MONGODB_URL);
+var mapper = new MongoMapper(process.env.MONGOLAB_URI);
```

Comment:
Best practice is to configure your deployment environment to suit your code, rather than change your code to expect a specific deployment environment. Using the generic `MONGODB_URL` variable lets us switch between Heroku/AWS, or MongoLab/RDS/Compose, without having to modify and re-deploy our code. Instead, we can just point `MONGODB_URL` to whatever we have configured in our environment.